cms/djangoapps/contentstore/utils.py

```diff
@@ -431,6 +431,17 @@ def get_course_outline_url(course_locator) -> str:
     return course_outline_url
 
 
+def get_library_content_picker_url(block_locator) -> str:
+    """
+    Gets course authoring microfrontend library content picker URL for the given parent block.
+    """
+    course_locator = block_locator.course_key
+    mfe_base_url = get_course_authoring_url(course_locator)
+    content_picker_url = f'{mfe_base_url}/component-picker?parentLocator={block_locator}'
```

Comment:
Do we even need to pass `parentLocator` anymore?

Reply:
We don't need it anymore, it was required when MFE was pushing the block to course. Now it only sends a message with selection details. I'll update it.